developers/weaviate/concepts/vector-index.md

```diff
@@ -52,24 +58,22 @@ To learn more about configuring the collection, see [this how-to page](../manage
 
 ### Distance metrics
 
-All of [the distance metrics](/developers/weaviate/config-refs/distances.md) can be used with any vector index type.
+All of [the distance metrics](/developers/weaviate/config-refs/distances.md), such as cosine similiarity, can be used with any vector index type.
 
 ## Set vector index type
 
 The index type can be specified per data collection via the [collection definition](/developers/weaviate/manage-data/collections.mdx#set-vector-index-type) settings.
 
 
-## Hierarchical Navigable Small World (HNSW)
+## What is Hierarchical Navigable Small World (HNSW)
 
-Weaviate's `hnsw` index is a [custom implementation](../more-resources/faq.md#q-does-weaviate-use-hnswlib) of the Hierarchical Navigable Small World ([HNSW](https://arxiv.org/abs/1603.09320)) algorithm that offers full [CRUD-support](https://db-engines.com/en/blog_post/87).
+Hierarchical Navigable Small World (HNSW) is an algorithm that works on multi-layered graphs. It is also an index type, and refers to vector indexes that are created using the HNSW algorithm. HNSW indexes provide benefits of very fast queries, but they are more costly when it comes to the building process (adding data with vectors).
```

Comment:
Consider:

HNSW indexes enable very fast queries, but rebuilding the index when you add new vectors can be resource intensive.

Reply:
included